### ISSUE TITLE
In your Supervised-Contrastive-Loss, why sim_matrix minus logit_max? 

### ISSUE THREAD

**Issue by yechenzhi** [alinlab/CSI#17]
https://github.com/alinlab/CSI/blob/21eb5000324c186b7f8fcd01b8d1a7aa2fae9d02/training/contrastive_loss.py#L61

**Comment by Jihoon-Tack** [comment]
Thank you for your interest in our paper!

We used the loss function from the original paper's implementation
- https://github.com/HobbitLong/SupContrast/blob/master/losses.py

The same question was raised in https://github.com/HobbitLong/SupContrast/issues/4

Thank you
